src/chainparams.cpp

```diff
@@ -393,7 +393,7 @@ class CRegTestParams : public CChainParams {
         consensus.nPowMaxAdjustDown = 0; // Turn off adjustment down
         consensus.nPowMaxAdjustUp = 0; // Turn off adjustment up
         consensus.nPowTargetSpacing = 2.5 * 60;
-        consensus.nPowAllowMinDifficultyBlocksFromHeight = 0;
+        consensus.nPowAllowMinDifficultyBlocksAfterHeight = 0;
```

Comment:
Notice that this doesn't allow the min-difficulty rule to apply to the regtest genesis block, but that's not a problem.

Reply:
It never can, since by definition we don't have a block before the genesis from which we can measure an interval that exceeds the 15-minute threshold.